docs/aws/bin_packing.md

```diff
@@ -0,0 +1,15 @@
+# Bin packing
```

Comment:
Put a TODO in this doc to formalize it into a design. Call out that this is a POC and preliminary, not a final design state.